wallet-rendering/spec.md

```diff
@@ -0,0 +1,200 @@
+  },
+  "background": {
+    "color": "#ff0000"
+  },
+  "text": {
+    "color": "#d4d400"
+  }
+}
+```
+:::
+
+ 
+An _Entity Style Descriptor_ ****must**** be an object composed of the following properties:
+
+- The object ****MAY**** contain a `thumbnail` property, and if present, its value ****MUST**** be an object with the following optional properties:
+    - The object ****MUST**** contain a `uri` property, and if present its value ****MUST**** be a valid URI string to an image resource.
+    - The object ****MAY**** contain an `alt` property, and if present its value ****MUST**** be a string that describes the alternate text for the logo image.
+- The object ****MAY**** contain a `hero` property, and if present, its value ****MUST**** be an object with the following optional properties:
+    - The object ****MUST**** contain a `uri` property, and if present its value ****MUST**** be a valid URI string to an image resource.
+    - The object ****MAY**** contain an `alt` property, and if present its value ****MUST**** be a string that describes the alternate text for the logo image.
+- The object ****MAY**** contain a `background` property, and if present, its value ****MUST**** be an object with the following optional properties:
+    - The object ****MAY**** contain a `color` property, and if present its value ****MUST**** be a HEX string color value (e.g. #000000).
+- The object ****MAY**** contain a `text` property, and if present, its value ****MUST**** be an object with the following optional properties:
+    - The object ****MAY**** contain a `color` property, and if present its value ****MUST**** be a HEX string color value (e.g. #000000).
+
+## Data Display
+
+A _Data Display Descriptor_ ****must**** be an object composed of the following properties:
+
+- The object ****MAY**** contain a `title` property, and if present, its value ****MUST**** be a [[ref:Display Mapping Object]]. User Agents ****SHOULD**** render the data in an area of UI that conveys the general title of the credential being rendered.
+- The object ****MAY**** contain a `subtitle` property, and if present, its value ****MUST**** be a [[ref:Display Mapping Object]]. User Agents ****SHOULD**** render the data in close proximity to the `title` value and ****SHOULD**** display the information in a way that is noticably less pronounced than that of the `title` value.
+- The object ****MAY**** contain a `description` property, and if present, its value ****MUST**** be a [[ref:Display Mapping Object]]. User Agents ****SHOULD**** render the data in an area of UI that is appropreate for verbose, descriptive textual data.
+- The object ****MAY**** contain a `properties` property, and if present, its value ****MUST**** be an array of [[ref:Labeled Display Mapping Objects]]. User Agents ****SHOULD**** render the data specified by each _Display Mapping Object_ in an area of UI that is appropreate for the rendering of a flat list of labeled values.
+
+### Display Mapping Object
```

Comment:
We should include both "Display Mapping Object" and "Labeled Display Mapping Object". The "Labeled" version is the same except it must have a "label" property, it's used for entries in the `ODO.display.properties` array

Reply:
Fine with adding in follow up PR